templates/deno/package.json

```
@@ -5,11 +5,12 @@
     "build": "remix build",
     "deploy": "deployctl deploy --prod --include=build,public --project=<your deno deploy project> ./build/index.js",
     "dev": "remix build && run-p dev:*",
-    "dev:deno": "cross-env NODE_ENV=development deno run --unstable --watch --allow-net --allow-read --allow-env ./build/index.js",
+    "dev:deno": "cross-env NODE_ENV=development deno run --unstable --watch --allow-net --allow-read --allow-env --no-check ./build/index.js",
     "dev:remix": "remix watch",
     "format": "deno fmt --ignore=node_modules,build,public/build",
     "lint": "deno lint --ignore=node_modules,build,public/build",
-    "start": "cross-env NODE_ENV=production deno run --unstable --allow-net --allow-read --allow-env ./build/index.js"
+    "start": "cross-env NODE_ENV=production deno run --unstable --allow-net --allow-read --allow-env --no-check ./build/index.js",
+    "typecheck": "deno check --import-map .vscode/resolve_npm_imports.json server.ts app/*.tsx app/*/*.tsx"
```

Does `deno check` support globs like `**/*.tsx`?

Also, we need to check `.ts` files too, not just `.tsx` files, so maybe we'd need something like:

```suggestion
    "typecheck": "deno check --import-map .vscode/resolve_npm_imports.json server.ts app/**/*.ts app/**/*.tsx"
```